Cangjie5.txt

```diff
@@ -59337,6 +59336,7 @@
 隮	nlyx
 𮥱	nlyxf
 𨼻	nlyxv
+𨽘	nlyxv
```

Comment:
It was `hyxv` before this PR, @Jackchows please double check if `nlyxv` is correct.